.github/workflows/masterbot.yml

```diff
@@ -0,0 +1,90 @@
+name: Master Robot
+on:
+  push:
+    branches:
+      - master
+    tags-ignore: ['*']
+jobs:
+  build:
```

Comment:
I wanted to extract all of these jobs into [reusable workflows](https://docs.github.com/en/actions/learn-github-actions/reusing-workflows). However, it seems like, unlike [composite actions](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action), we cannot call a reusable workflow that is part of the HEAD commit.